.github/workflows/rpm-build.yml

```diff
@@ -0,0 +1,17 @@
+name: rpm-build
+
+on:
+  pull_request:
+    branches:
+      - master
+      - rhel-8
+
+jobs:
+  rpm-build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run rpm-build test
+        run: make test_rpmbuild
```

Comment:
Does that invoke the container by itself? I don't see a `container:` in the workflow, but then again Travis didn't do that either.